internal/resources/resources_test.go

```diff
@@ -0,0 +1,258 @@
+package resources
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-logr/logr"
+	"github.com/kyma-project/api-gateway/apis/gateway/v1beta1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	networkingv1alpha3 "istio.io/client-go/pkg/apis/networking/v1alpha3"
+	networkingv1beta1 "istio.io/client-go/pkg/apis/networking/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var sc *runtime.Scheme
+
+var _ = Describe("Resources", func() {
+	sc = runtime.NewScheme()
+	Expect(v1beta1.AddToScheme(sc)).To(Succeed())
+	Expect(networkingv1alpha3.AddToScheme(sc)).To(Succeed())
+	Expect(networkingv1beta1.AddToScheme(sc)).To(Succeed())
+
+	DescribeTable("FindUserCreatedIstioResourcesDescribe", func(ctx context.Context, logger logr.Logger, c client.Client, configuration resourceFinderConfiguration, conditionResult bool, want []Resource, wantErr bool) {
```

Comment:
I think this should be renamed.
```suggestion
	DescribeTable("FindUserCreatedResourcesDescribe", func(ctx context.Context, logger logr.Logger, c client.Client, configuration resourceFinderConfiguration, conditionResult bool, want []Resource, wantErr bool) {
```